docs/how_to/install.rst

```diff
@@ -253,7 +253,7 @@ Further required configuration
 URLs
 ====
 
-In the project's ``urls.py``, add ``url(r'^', include('cms.urls'))`` to the ``urlpatterns`` list. It should come after
+In the project's ``urls.py``, add ``re_path(r'^', include('cms.urls'))`` to the ``urlpatterns`` list. It should come after
```

Comment:
Thanks for your pull request.

Django 1.11 doesn't support `re_path` - it was introduced in Django 2.0.

We should keep documentation current and have support for 1.11 as well. My suggestion is just to add an additional note about the new way of using URLs since Django 2.0+.

Reply:
Reverted the change and added a note as suggested.